wand/api.py

```diff
@@ -118,6 +140,12 @@ def load_library():
     elif ctypes.sizeof(ctypes.c_ulonglong) == ctypes.sizeof(ctypes.c_void_p):
         ctypes.c_ssize_t = ctypes.c_longlong
 
+# FIXME: On 64-bit Windows, passing in a pointer using ctypes.c_void_p will
+# occasionally fail with an OverflowError for no apparent reason. I have not
+# yet found a suitable fix for this problem
+# 
+# One proposed solution is subclassing ctypes.c_void_p, but that breaks the
+# assert statements.
```

Comment:
A subclass is always `False` in a boolean expression, even if that pointer is NULL. But I just found out there was a signature missing in `api.py`. With that declaration I no longer get that error.

But that being said, I saw the error before when trying to call `MagickSetFormat` but I'm not seeing that error anymore. I'll let you know if it comes back.